service/src/main/java/uk/gov/hmcts/reform/fpl/service/ManageHearingsService.java

```diff
@@ -77,15 +80,27 @@ public DynamicList asDynamicList(List<Element<HearingBooking>> hearingBooking, U
     }
 
     public UUID adjournAndReListHearing(CaseData caseData, UUID hearingId, HearingBooking hearingToBeReListed) {
-        Element<HearingBooking> adjournedBooking = adjourn(caseData, hearingId, ADJOURNED_AND_RE_LISTED);
+        Element<HearingBooking> adjournedBooking = cancelHearing(caseData, hearingId, ADJOURNED_AND_RE_LISTED);
         Element<HearingBooking> reListedBooking = reList(caseData, hearingToBeReListed);
 
         reassignDocumentsBundle(caseData, adjournedBooking, reListedBooking);
         return reListedBooking.getId();
     }
 
+    public UUID vacateAndReListHearing(CaseData caseData, UUID hearingId, HearingBooking hearingToBeReListed) {
```

Comment:
minor could be 
`
    public UUID adjournAndReListHearing(CaseData caseData, UUID hearingId, HearingBooking hearingToBeReListed) {
        return cancelAndReListHearing(caseData, hearingId, hearingToBeReListed, ADJOURNED_AND_RE_LISTED);
    }

    public UUID vacateAndReListHearing(CaseData caseData, UUID hearingId, HearingBooking hearingToBeReListed) {
        return cancelAndReListHearing(caseData, hearingId, hearingToBeReListed, VACATED_AND_RE_LISTED);
    }

    private UUID cancelAndReListHearing(CaseData caseData, UUID hearingId, HearingBooking hearingToBeReListed, HearingStatus status){
        Element<HearingBooking> vacatedBooking = cancelHearing(caseData, hearingId, status);
        Element<HearingBooking> reListedBooking = reList(caseData, hearingToBeReListed);

        reassignDocumentsBundle(caseData, vacatedBooking, reListedBooking);
        return reListedBooking.getId();
    }
`

but maybe your version is easier to read